data/src/main/java/com/mashup/gabbangzip/sharedalbum/data/base/PicResponse.kt

```diff
@@ -21,8 +21,19 @@ data class PicErrorResponse(
     val message: String,
 )
 
+data class PicApiException(
+    val errorResponse: PicErrorResponse?,
+) : RuntimeException() {
+    override val message: String
+        get() = errorResponse?.message.orEmpty()
+}
+
 suspend fun <T> callApi(
     execute: suspend () -> PicResponse<T>,
 ): T {
-    return execute().data ?: throw IllegalStateException()
+    return runCatching {
+        execute().data!!
```

Comment:
```suggestion
        execute().data ?: throw NoSuchFieldException()
```
API 상에서 data 가 null 로 내려오는 상황이 발생하지 않는다고 가정할 순 없으니, data 가 null 이면 `NoSuchFieldException` 으로 해당 field 값이 존재하지 않는다고 보내서 PicApiErrorParser 로 자연스럽게 넘어가게 하는건 어떨까

Reply:
서버에서 isSuccess가 true이면 data가 항상 있어서.. 없으면 어차피 HttpException에서 먼저 걸려서 자연스럽게 파서로 넘어가지 않을까?

Reply:
[ae1be98](https://github.com/mash-up-kr/gabbangzip-Android/pull/324/commits/ae1be98efc26c746e3f9d80e81cd556c2bd2d1cb) 반영완!

Reply:
아 그럴수 있겠네..? 저게 반영되지 않았어도 크게 문제는 없었겠다..
확인 고마워~~